less/mosaico/article.less

```diff
@@ -206,7 +206,31 @@
     margin-top: 30px;
     margin-bottom: 10px;
   }
-
+  &__related {
+    display: block;
+    margin: 30px auto;
+    padding: 0 30px 0 15px;
+    ul {
+      list-style: none;
+      margin-top: 5px;
+      li {
+        border-left: 4px solid @hbl-orange; // TODO: deal with other site's colours
```

Comment:
We should probably add some margin to the `li` elements, as per the layout, so we have that small gap between articles

Reply:
I already had `padding-bottom` for them, I changed it to `margin-bottom`. No material difference in this case.